WordPress/Sniffs/Theme/NoHardCodedUrlsSniff.php

```diff
@@ -0,0 +1,87 @@
+<?php
+/**
+ * WordPress Coding Standard.
+ *
+ * @category PHP
+ * @package  PHP_CodeSniffer
+ * @link     https://make.wordpress.org/core/handbook/best-practices/coding-standards/
+ */
+
+/**
+ ** WordPress_Sniffs_Theme_NoHardCodedUrlsSniff Class Doc Comment
```

Comment:
Could you remove the extra * and "Class Doc Comment" but add a full stop at the end.